packages/scaffolding-cli/templates/src/ssr/pages/_app.tsx

```diff
@@ -44,17 +50,21 @@ class _App extends App<AppWithStore> {
 
     render() {
         const {Component, pageProps, store} = this.props
+        const props = {
+            ...pageProps,
+            user: this.state,
+        }
         return (
             <>
                 <Head>
-                    <title>Amido stacks</title>
+                    <title>Amido stacks SSR</title>
```

Comment:
Can we change this to `Amido Stacks SSR` ? Sorry! Captilisation.

Reply:
Done